src/net/http.rs

```diff
@@ -1674,6 +1675,19 @@ impl HttpRequestType {
         !no_proof
     }
 
+    /// Check whether the given option query string sets use_latest_tip=1 (setting use_latest_tip to true).
+    /// Defaults to false.
+    fn get_use_latest_chain_tip(query: Option<&str>) -> bool {
+        if let Some(query_string) = query {
+            form_urlencoded::parse(query_string.as_bytes())
+                .find(|(key, _v)| key == "use_latest_tip")
+                .map(|(_k, value)| value == "1" || value == "true")
```

Comment:
Do we gain anything concrete by accepting either/or `1` or `true`?  Might as well just pick one or the other if not.

Reply:
Yep, if we want to adhere to OpenAPI schema and general API design guidelines, this should be a boolean rather than an `number` or `string`. I suggested also allowing a number for both consistency purposes (some existing endpoints use a number rather than a boolean), as well as the robustness principal (permissive in what is accepted) because this isn't really a security-sensitive feature. 

In the stacks-blockchain-api, similar boolean query params are pretty liberal in what is accepted, because it's not entirely unreasonable for someone to do any of the following:
`curl http://example.com/?use_latest_tip=true`,
`curl http://example.com/?use_latest_tip=1`, or even
`curl http://example.com/?use_latest_tip`. 

But, if there are reservations around accepting multiple forms of query param input, it should be a proper boolean.

Reply:
What's here is fine.